gui.py

```diff
@@ -270,6 +276,27 @@ def get_local_config(self):
         devices, shifts = self.get_device_details()
         return config_template.format(self.textbox_erpnext_api_key.text(), self.textbox_erpnext_api_secret.text(), self.textbox_erpnext_url.text(), self.textbox_pull_frequency.text(), formated_date, json.dumps(devices), json.dumps(shifts))
 
+    def get_running_status(self):
+        running_status = []
+        found = False
+        with open('/'.join([config.LOGS_DIRECTORY])+'/logs.log', 'r') as f:
+            for line in f:
+                logdate = datetime.datetime.strptime(line.split(',')[0], '%Y-%m-%d %H:%M:%S')
+                if logdate >= datetime.datetime.strptime(self.service_start_time.text().split('.')[0] , '%Y-%m-%d %H:%M:%S'):
+                    running_status.append(line)
+        with open('/'.join([config.LOGS_DIRECTORY])+'/error.log', 'r') as f:
+            for line in f:
+                start_date = datetime.datetime.strptime(self.service_start_time.text().split('.')[0] , '%Y-%m-%d %H:%M:%S')
+                if start_date.strftime('%Y-%m-%d') in line:
+                    logdate = datetime.datetime.strptime(line.split(',')[0], '%Y-%m-%d %H:%M:%S')

```

Comment:
@KaviyaPeriyasamy, this would throw an exception and crash if the expected formatted it of datetime is not found... Have you tested it with a garbage value in the logfile? 

you can use a function like this : https://github.com/frappe/biometric-attendance-sync-tool/blob/0d0ffe369dbc3312eb5efbceb0d69180d3bde5c8/erpnext_sync.py#L283

Reply:
@karthikeyan5 I updated the requested changes. Can you review this commit a8a7ce1222c82f7c5f5b005c7843d94b2fbcd158